app/models/oseq_flowcell_run.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
```

Comment:
Hi, the migration & sql stuff look fine, just struggling on this bit - what's the reason for the new model? 
Thanks

Reply:
There are two versions of traction. Old traction is still being used for non covid work and does not do multiplexing. This only has a single library per flowcell. If we used the same model it would break sending messages for existing traction. So we created a new model to handle new messages for new traction. Once we switch off old traction we will merge it again. Hope that makes sense?

Reply:
OK yes I think so, thanks.